boot/SPDX.LicenseExceptionId.template.hs

```diff
@@ -39,6 +40,10 @@ instance Binary LicenseExceptionId where
         then fail "Too large LicenseExceptionId tag"
         else return (toEnum (fromIntegral i))
 
+-- note: remember to bump version each time the definition changes
```

Comment:
This seems to defeat the purpose of this patch. Any way this datatype can be included in the structure hash too?

Reply:
Manual `Binary` instance => Manual `Structured` instance.

Yet, here `Binary` instance is generated by templating, so yes, we should generate `Structured` instance too. I'll take the *license list version* and use that.

Good catch.